helix-term/src/ui/editor.rs

```diff
@@ -810,6 +814,29 @@ impl Component for EditorView {
                 status_msg,
                 style,
             );
+        } else {
```

Comment:
Should we put this in the else block since both can be there at the same time?

Reply:
I put it in else since `status_msg` might be long enough to overlap with key display. I think calculating the width and deciding based on available space is better now that you mention it.